pages/docs/tutorials/kafka/managing-schemas-using-schema-registry.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Managing schemas using Schema Registry
+description: The guide explains how you can manage schemas using Schema Registry.
+weight: 50
+---
+
+## Introduction
+You will learn how to manage schemas with Schema Registry. Managing schemas will enable us to ensure a consistent data exchange across distributed systems, thus minimizing errors.
+
+## Background context
+The need for schema management has become increasingly prevalent to handle the evolving complexity of modern Event-Driven Architecture. A Schema Registry is a centralized service that stores and maintains the schemas for data exchanged between various components of a modern distributed system. Validating data being exchanged helps in maintaining data consistency and compatibility. 
+
+While several Schema Registry implementations exist, you will use the Apicurio Registry in this tutorial. Apicurio Registry is a popular open-source Schema Registry implementation that supports multiple serialization formats and facilitates schema management for diverse data in distributed systems. You will be using the Apicurio Registry in combination with Avro, which is a language-neutral data serialization system.
+
+## Installation guide
+1. Install [Docker](https://docs.docker.com/engine/install/) from the official website.
+
+## Steps
+1. Run Apicurio Registry Docker Image in a Docker container by using the command:
+    <CodeBlock language="bash">
+    {`docker run -it -p 8080:8080 apicurio/apicurio-registry-mem:2.5.7.Final`}
+    </CodeBlock>
+2. Once your Apicurio Registry is running, you can upload the schema in another terminal window. Create an Avro Schema Artifact using the following command:
+<CodeBlock>
+curl \
+http://localhost:8080/apis/registry/v2/groups/my-group/artifacts \
+-X POST  \
+-H "Content-Type: application/json; artifactType=AVRO" \
+-H "X-Registry-ArtifactId: UserSignedUp" \
+--data @- << EOF
+{
+  "type": "record",
+  "name": "UserSignedUp",
+  "namespace": "com.company",
+  "doc": "User sign-up information",
+  "fields": [
+    {
+      "name": "userId",
+      "type": "int"
+    },
+    {
+      "name": "userEmail",
+      "type": "string"
+    }
+  ]
+}
+EOF
+</CodeBlock>
+3. You can download the Avro schema by visiting the URL below and replacing {UserSignedUp} with the id of the schema used in the previous step.
+http://localhost:8080/apis/registry/v2/groups/my-group/artifacts/{UserSignedUp}
+4. An alternative to having the schema in a separate file as in (Joy's tutorial), would be to store it in the registry. After uploading your Avro schema, remove the schema from your document, and replace it with the following:
```

Comment:
you definitely should not write `Joy's tutorial` in official docs.

please look at 2 PRs from Joy's, where he has 2 tutorials: `main` and the `followup` that refers to the `main` one in a very nice way, showing previous document from `main` tutorial, and then pointing the part that is modified in `followup` tutorial. Please consult with him and update here

Reply:
I will discuss with Joy about this.

Reply:
ok, looking forward

here you have example document with reference to apicurio uploaded schema - works well

```yaml
asyncapi: 3.0.0
info:
  title: User Signup API
  version: 1.0.0
  description: The API notifies you whenever a new user signs up in the application.
servers:
  kafkaServer:
    host: test.mykafkacluster.org:8092
    description: Kafka Server
    protocol: kafka
operations:
  onUserSignedUp:
    action: receive
    channel:
      $ref: '#/channels/userSignedUp'
channels:
  userSignedUp:
    description: This channel contains a message per each user who signs up in our application.
    address: user_signedup
    messages:
      userSignedUp:
        $ref: '#/components/messages/userSignedUp'
components:
  messages:
    userSignedUp:
      payload:
        schemaFormat: 'application/vnd.apache.avro+json;version=1.9.0'
        schema:
          $ref: http://localhost:8080/apis/registry/v2/groups/my-group/artifacts/UserSignedUp
```

Reply:
Yes @Arya-Gupta, you should not include Joy's name in documentation because this is not a personal blog post.  You should be referring to the tutorial by name clearly outlined in Joy's tutorial PR: https://github.com/asyncapi/website/pull/2329/files#diff-b418e5b2374213ec5e0acd798a9a87b653513a33c1b0f042e7e109aa15e30d38. 

EXAMPLE: 
> In the previous tutorial, **Describe Kafka message payload using Avro Schema,** you learned ABC. In this tutorial, you will use that Avro schema to..."

Reply:
I have fixed the link, and added the snippet after the last step in my tutorial.